src/traminformation.py

```diff
@@ -25,7 +24,7 @@ class TramInformation:
     def minutes_till_departure(self, now: datetime.datetime) -> int:
         now: datetime = now or datetime.datetime.utcnow()
         seconds: float = (self.eta - now).total_seconds()
-        return round(seconds / 60)
+        return abs(round(seconds / 60))
```

Comment:
And why are we doing this? Wouldn't it show a tram that should be leaving as one that should be arriving in a minute?

Reply:
Ok, when I read your comment, I realized it's not a solution for -1 minute until departure because it would change for one minute